packages/@office-iss/react-native-win32-tester/package.json

```diff
@@ -25,12 +25,11 @@
     "@rnw-scripts/ts-config": "2.0.2",
     "@types/node": "^14.14.22",
     "eslint": "^7.32.0",
-    "just-scripts": "^1.3.3",
```

Comment:
Removal of `just-scripts` is because we use `@rnw-scripts/just-task` directly, which dictates this. Some packages still need it though because they customize the default build, lint, or other tasks.